internal/pkg/manifest/validate_env.go

```diff
@@ -182,3 +185,13 @@ func (o publicHTTPConfig) Validate() error {
 	}
 	return nil
 }
+
+// Validate returns nil if privateHTTPConfig is configured correctly.
+func (o privateHTTPConfig) Validate() error {
+	for idx, certARN := range o.Certificates {
+		if _, err := arn.Parse(certARN); err != nil {
+			return fmt.Errorf(`validate "certificates[%d]": %w`, idx, err)
```

Comment:
I still think both this and https://github.com/aws/copilot-cli/pull/3591/files#diff-80023f967d4bab363b7e0d95b3ff0467441beb4da4da9c49fbaf918cfb5c26b3L180 should be "parse" instead of "validate" because this function is to validate but the public function we call for catching the error is `arn.Parse`